modules/schema_validation.lua

```diff
@@ -85,6 +85,11 @@ local function compare(schema, function_id, msgType, user_data, mandatory_check)
     local retval= {}
     local class_, short_name
     if (name == nil ) then return retval end
+
+    if (name == "WrongFunctionName") then
+      name = "GenericResponse"
```

Comment:
@LuxoftAKutsan Used in https://github.com/AMikhniev/sdl_atf/commit/5952cb79e52d8f9a30b6d26fa2fbd2a8d5227051